admin/sql/test/bb-test-data.sql

```diff
@@ -9,3 +11,35 @@ INSERT INTO relationship_set (id) VALUES (99999999);
 INSERT INTO relationship_set__relationship (set_id, relationship_id) 
      VALUES (99999999,
             99999999);
+
+
+-- Sample Data for testing author credits for edition groups
+
+UPDATE edition_group_data
+   SET author_credit_id = NULL
+  FROM edition_group_revision as egr, edition_group_header as egh
+ WHERE egr.data_id = edition_group_data.id
+   AND egh.master_revision_id = egr.id
+   AND egh.bbid = 'fd84cf1f-b288-4ea2-8e05-41257764fa6b';
+
+
+INSERT INTO author_credit (id, author_count, ref_count)
+     VALUES (99999999,
+            1,
+            0);
+
+
+INSERT INTO author_credit_name (author_credit_id, position, author_bbid, name, join_phrase )
+    VALUES (99999999,
+            0,
+            'e5c4e68b-bfce-4c77-9ca2-0f0a2d4d09f0',
+            'Test Author',
+            'Test Join Phrase');
```

Comment:
for your information, normally a join phrase only exists if there are more than 1 name in the credit. It's so that you can do "Test Author **and** Someone Else" where __and__ is the join phrase. However, this is fine